pkg/prometheus/resource_selector_test.go

```diff
@@ -3393,6 +3393,29 @@ func TestSelectScrapeConfigs(t *testing.T) {
 			},
 			selected: false,
 		},
+		{
+			scenario: "OVHCloud SD config",
```

Comment:
It need not be done in this same PR, but tests with faulty (non-existent) SecretKeySelectors might also be a good addition.

Reply:
Got it.